core/src/main/java/com/linecorp/armeria/internal/server/annotation/AnnotatedValueResolver.java

```diff
@@ -90,18 +90,43 @@
 final class AnnotatedValueResolver {
     private static final Logger logger = LoggerFactory.getLogger(AnnotatedValueResolver.class);
 
-    private static final List<RequestObjectResolver> defaultRequestConverters = ImmutableList.of(
-            (resolverContext, expectedResultType, expectedParameterizedResultType, beanFactoryId) -> {
-                final AnnotatedBeanFactory<?> factory = AnnotatedBeanFactoryRegistry.find(beanFactoryId);
-                if (factory == null) {
-                    return RequestConverterFunction.fallthrough();
-                } else {
-                    return factory.create(resolverContext);
-                }
-            },
-            RequestObjectResolver.of(new JacksonRequestConverterFunction()),
-            RequestObjectResolver.of(new StringRequestConverterFunction()),
-            RequestObjectResolver.of(new ByteArrayRequestConverterFunction()));
+    @Nullable
+    private static final RequestConverterFunction protobufRequestConverterFunction;
+
+    static {
+        RequestConverterFunction converterFunction;
+        try {
+            final Class<?> protobufConverterClass =
+                    Class.forName("com.linecorp.armeria.server.grpc.ProtobufRequestConverterFunction");
```

Comment:
We use SPI to find `ResponseConverterFunction` implementations.
Shouldn't we use the same approach for `RequestConverterFunction`?

Reply:
That's a good suggestion. Let me add a SPI provider.